waltz-ng/client/physical-flows/physical-flow-registration.html

```diff
@@ -81,6 +81,48 @@
 
         </div>
 
+
+        <div ng-if="ctrl.visibility.editor == '' && ctrl.similarFlows.length > 0">
+            <div class="row" ng-hide="ctrl.visibility.similarFlows">
```

Comment:
re earlier comment about `visibility.similarFlows` vs `ctrl.similarFlows` we have both flavours of the test here....

Don't think the extra derived flag is adding a lot.